Controller/ElasticsearchController.php

```diff
@@ -616,10 +616,6 @@ public function searchApiAction(Request $request)
     }
 
     /**
-     * @param Request $request
-     *
-     * @param JobService $jobService
-     * @return RedirectResponse
      * @Route("/search/export/{contentType}", name="emsco_search_export", methods={"POST"})
```

Comment:
Keep return as it is not defined in the function definition

Reply:
Rollbacked